db/seeds.d/16-mail_notifications.rb

```diff
@@ -49,11 +50,14 @@
     :description        => N_('A test message to check the email configuration is working'),
     :mailer             => 'UserMailer',
     :method             => 'tester',
-    :subscription_type  => false
+    :subscriptable      => false,
+    :subscription_type  => nil
   }
 ]
 
 notifications.each do |notification|
-  MailNotification.create(notification) if MailNotification.where(:name => notification[:name]).blank?
+  mail = MailNotification.where(:name => notification[:name]).first_or_initialize(notification)
+  mail.new_record? ? mail.save : mail.assign_attributes(notification)
+  mail.save if mail.subscriptable_changed?
```

Comment:
The flow in this block is a bit hard to read, since this line is only going to be valid when `!mail.new_record?`.  It might be longer in lines, but it may be clearer to separate the new record from the old record flow, e.g.

<pre>
if (mail = MailMailNotification.find_by_name(notification[:name]))
  mail.attributes = notification
  mail.save if mail.changes.present?
else
  MailNotification.create(notification)
end
</pre>

Also it's missing error handling at the moment, so if this .save fails, it won't be picked up.  Look at the usage of `format_errors` in other seed files and then raise an error if .create or .save returns false.